scripts/version/create-snippet.js

```diff
@@ -4,17 +4,22 @@ const path = require('path');
 const { snippet } = require('../templates/browser-snippet.template');
 const { getName, getVersion } = require('../utils');
 const babel = require('@babel/core');
+const yargs = require('yargs/yargs')
+const { hideBin } = require('yargs/helpers')
+const argv = yargs(hideBin(process.argv)).argv
 
 // Setup input
+const isGTM =  argv.isGTM === 'true';
 const inputDir = 'lib/scripts';
-const inputFile = 'amplitude-min.js';
+const inputFile = isGTM ? 'amplitude-gtm-min.js' : 'amplitude-min.js';
 const inputPath = path.join(process.cwd(), inputDir, inputFile);
 
 // Setup output
 const outputDir = 'generated/';
-const outputFile = 'amplitude-snippet.js';
+const outputFile = isGTM ? 'amplitude-gtm-snippet.js' : 'amplitude-snippet.js';
 const outputPath = path.join(process.cwd(), outputDir, outputFile);
 
+const globalVar = isGTM ? 'amplitudeGTM' : 'amplitude';
```

Comment:
nit: for more flexible snippet creation via CL I would suggest exposing `inputFile`, `ouputFile`, and `globalVar` args rather than `isGTM`.

```
const inputFile = argv.inputFile ? 'amplitude-min.js';
const outputFile = argv.outputFile ? 'amplitude-snippet.js';
const globalVar = argv.globalVar ? : 'amplitude';
```